src/common.h

```diff
@@ -12,7 +12,6 @@
 
 #include <nan.h>
 
-#include <iostream>
 #include <string>
```

Comment:
```suggestion
#include <iostream>
#include <string>
```
Hi @Qard 👋🏼  The import should be reverted. The code doesn't compile ATM

Reply:
```cc
../src/common.cc:18:8: error: no member named 'cerr' in namespace 'std'
   18 |   std::cerr << "% " << str.c_str() << std::endl;
      |   ~~~~~^
```

Reply:
Odd. Seemed to work for me on macOS and it complained that it was unused. I've restored it.